core/src/main/scala/io/delta/sql/parser/DeltaSqlParser.scala

```diff
@@ -201,6 +201,10 @@ class DeltaSqlAstBuilder extends DeltaSqlBaseBaseVisitor[AnyRef] {
       Option(ctx.table).map(visitTableIdentifier))
   }
 
+  override def visitShowCreateTable(ctx: DeltaSqlBaseParser.ShowCreateTableContext): LogicalPlan = withOrigin(ctx) {
+    ShowCreateTableCommand(Option(ctx.table).map(visitTableIdentifier))
```

Comment:
Similar to `show columns`, could you detect whether this is a Delta table or not here? If it's not a Delta table, you can return `null` here to make the code fallback to Spark SQL parser.

Reply:
Yes. Will make this change.